src/internet_identity/tests/integration/activity_stats/authn_methods.rs

```diff
@@ -250,7 +250,7 @@ fn should_report_active_openid_authn_methods() {
 
     // OpenID test data and fetch Google certs (mock)
     let (jwt, salt, _claims, test_time, test_principal, _test_authn_method) =
-        openid::openid_test_data();
+        openid::openid_google_test_data();
```

Comment:
Should there be a comment to explain why Google is picked for the test?

Reply:
Good point, I'll change the comment above